site/_data/sidebar.yaml

```diff
@@ -93,6 +93,8 @@ groups:
     endpoint: non-persistent-topics
   - title: Partitioned topics
     endpoint: partitioned-topics
+  - title: Encryption
+    endpoint: Encryption
```

Comment:
Perhaps,this will be dead link since the URL path become `/docs/latest/admin-api/Encryption`, not `/docs/latest/advanced/Encryption`.

Reply:
Yes, good catch. Fixed.